packages/server/graphql/types/AgendaItem.ts

```diff
@@ -32,6 +32,10 @@ const AgendaItem = new GraphQLObjectType<IAgendaItem, GQLContext>({
       description: 'true if the agenda item has not been processed or deleted',
       resolve: ({isActive}) => !!isActive
     },
+    pinned: {
+      type: new GraphQLNonNull(GraphQLBoolean),
+      description: 'True if the agenda item has been pinned'
```

Comment:
type is nonnull, but pinned will be null for existing & previous meetings. might want to resolve to false if undefined

Reply:
Thanks! Updated